packages/trpc/server/routers/viewer/apps.tsx

```diff
@@ -268,4 +268,13 @@ export const appsRouter = router({
         },
       });
     }),
+  checkForGCal: authedProcedure.query(async ({ ctx }) => {
+    const gCalPresent = ctx.prisma.credential.findFirst({
```

Comment:
Was always returning true since an unawaited promise is truth.

```suggestion
    const gCalPresent = await ctx.prisma.credential.findFirst({
```